Marlin/pins.h

```diff
@@ -172,6 +172,8 @@
   #include "pins_MKS_13.h"
 #elif MB(SAINSMART_2IN1)
   #include "pins_SAINSMART_2IN1.h"
+#elif MB(PICA) || MB(PICA_REVB)
```

Comment:
Usually there's one pins_xxxx.h for each motherboard.

In this case the pins_PICA_REVB.h would consist of your header and the following lines in this order:
```
#define BOARD_NAME "PICA_REVB" 
#include "pins_PICA.h"
```

You'll need to change the **# if MB(....** statement in pins_PICA.h to accommodate this change.

You may get a request to change the names of the boards so that it's easy to determine which is older just by looking at the names.  I suggest:
- pins_PICA_REVC.h & pins_PICA_REVB.h in case there should be a rev D in the future
- pins_PICA.h & pins_PICA_OLD.h style has also been used

I know you'll be able to find different naming conventions and the use of **# if MB(....** in other pins_XXXX.h files but it appears those have fallen out of favor.